.gitignore

```diff
@@ -43,6 +43,7 @@ coverage
 
 # Compiled binary addons (http://nodejs.org/api/addons.html)
 */dist
+*/gpx_*
```

Comment:
Somehow a windows binary got committed to the root directory... not sure how 🤷‍♂️ .